src/pages/Compare/index.spec.tsx

```diff
@@ -1108,15 +1164,24 @@ describe(__filename, () => {
     const setUpForPreloadAndRender = ({
       buildTree = true,
       currentFile = 'currentFile.json',
-      headVersionId = 5,
+      currentFileStatus = 'M',
+      headVersionId = nextUniqueId(),
       nextModifiedFile = 'modifiedFile.json',
+      nextModifiedFileStatus = 'M',
       ...params
-    }) => {
+    }: {
+      buildTree?: boolean;
```

Comment:
Some of these, which existed before, weren't specified before, e.g., `buildTree` and `currentFile`. Why are they needed now?

Reply:
I only needs `curentFileStatus?:VersionEntryStatus;` and `nextModifiedFileStatus?:VerionEntryStatus`, so that I do not have to cast a string ('M', 'D') to `VersionEntryStatus` in the function to make TSC happy. However I do not have a way to add those two only,  so I added the type of first argument (which includes all, like `buildTree` and `currentFile`...).